react/features/follow-me/middleware.js

```diff
@@ -101,14 +112,36 @@ function _onFollowMeCommand(attributes = {}, id, store) {
         return;
     }
 
+    if (!state['features/follow-me'].followMeActive) {
+        store.dispatch(setFollowMeActive(true, id));
+    }
+
+    // just a command that follow me was turned off
+    if (attributes.off) {
+        store.dispatch(setFollowMeActive(false));
+
+        return;
+    }
+
+    const oldState = state['features/follow-me'].followMeState || {};
+
+    store.dispatch(setFollowMeState(attributes));
```

Comment:
attributes is the cached follow me state?

Reply:
Yes.